File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        arr = [0]*(len(nums)+1)
        for i in range(len(nums)):
            arr[nums[i]] = 1
        for i in range(len(arr)):
            if arr[i]==0:
                return i
        return None

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int arr[] = new int[nums.length+1];
        for(int num : nums)
            arr[num] = 1;
        for(int i=0;i<arr.length;i++)
            if(arr[i]==0)
                return i;
        return -1;
    }
}